python/torch_mlir/extras/fx_importer.py

```diff
@@ -1485,6 +1585,74 @@ def _import_torch_op_overload(
         for i, value in enumerate(operation.results):
             self.bind_node_value(node, value, i)
 
+    def _import_shape_symbols_with_guards(
+        self, loc: Location, symbolic_guards: Dict[str, RangeConstraint]
+    ):
+        for symbol, constraints in symbolic_guards.items():
+            # Create torch.sym_int ops
+            operation = Operation.create(
+                name="torch.symbolic_int",
+                attributes={
+                    "symbol_name": StringAttr.get(symbol),
+                    "min_val": self._cc.integer_attr(constraints.min_val, 64),
+                    "max_val": self._cc.integer_attr(constraints.max_val, 64),
+                },
+                results=[self._cc.torch_int_type],
+                loc=loc,
+            )
+            self.bind_symbol_value(symbol, operation.result)
+
+    def _create_bind_symbolic_shape_ops(self, loc: Location, node: torch_fx.Node):
+        node_val = node.meta.get("val")
+        if (node_val is not None) and isinstance(node_val, TorchFakeTensor):
+            # Only create bind ops if the shapes contain symbolic sizes
+            if node_val._has_symbolic_sizes_strides:
```

Comment:
Could you add some commentary either pointing to docs or having an inline explanation about the types involved in _has_symbolic_sizes_strides? I think I roughly understand it but would have trouble modifying it without a lot of digging.

Reply:
`_has_symbolic_sizes_strides` is a bool attribute. I've added an inline comment (couldn't find a doc upstream).